inst/scripts/example_ctree_method.R

```diff
@@ -0,0 +1,40 @@
+library(shapr)
```

Comment:
I am surprised that this script does not include any categorical features to show how that works with shapr? Is that in another script that was submitted earlier?

Reply:
There are example(s) of this in the vignette. Not use if it has ever been in a single script like this. Fixing by adding modified example from vignette.

Reply:
You're right, I can't find an example.  But thanks.

Reply:
For some reason that part of the code does not show up when showing all commits under "files changes". You should see it if you select "changes from last review" in the upper when in the "file changes" view.